ember-cli-build.js

```diff
@@ -0,0 +1,154 @@
+                    <script>Raven.config("${config.sentryDSN}", {}).install();</script>`
+            },
+            cdn: {
+                enabled: useCdn,
+                content: `
+                    <script src="//cdnjs.cloudflare.com/ajax/libs/jquery/2.2.4/jquery.min.js"></script>
+                    <script src="//cdnjs.cloudflare.com/ajax/libs/ember.js/2.7.1/ember.prod.js"></script>`
+            },
+            assets: {
+                enabled: true
+            }
+        },
+        postcssOptions: {
+            compile: {
+                enabled: false
+            },
+            filter: {
+                enabled: true,
+                plugins: [{
+                    module: require('autoprefixer'),
+                    options: {
+                        browsers: ['last 4 versions'],
+                        cascade: false
+                    }
+                }, {
+                    // Wrap progid declarations with double-quotes
+                    module: require('postcss').plugin('progid-wrapper', () => {
+                        return css =>
+                            css.walkDecls(declaration => {
+                                if (declaration.value.startsWith('progid')) {
+                                    return declaration.value = `"${declaration.value}"`;
+                                }
+                            });
+                    })
+                }]
+            }
+        },
+        // bable options included to fix issue with testing discover controller
+        // http://stackoverflow.com/questions/32231773/ember-tests-passing-in-chrome-not-in-phantomjs
+        'ember-cli-babel': {
+            includePolyfill: true
+        },
+    });
+
+    // Use `app.import` to add additional libraries to the generated
+    // output files.
+    //
+    // If you need to use different assets in different
+    // environments, specify an object as the first parameter. That
+    // object's keys should be the environment name and the values
+    // should be the asset to use in that environment.
+    //
+    // If the library that you are including contains AMD or ES6
+    // modules that you would like to import into your application
+    // please specify an object with the list of modules as keys
+    // along with the exports of each module as its value.
+
+    // osf-style
+    app.import(path.join(app.bowerDirectory, 'osf-style/vendor/prism/prism.css'));
+    app.import(path.join(app.bowerDirectory, 'osf-style/page.css'));
+    app.import(path.join(app.bowerDirectory, 'osf-style/css/base.css'));
+    app.import(path.join(app.bowerDirectory, 'loaders.css/loaders.min.css'));
+
+
+    app.import(path.join(app.bowerDirectory, 'osf-style/img/cos-white2.png'), {
+        destDir: 'img'
+    });
+
+    app.import(path.join(app.bowerDirectory, 'jquery.tagsinput/src/jquery.tagsinput.js'));
+    app.import(path.join(app.bowerDirectory, 'c3/c3.js'));
```

Comment:
I don't think we'll be using c3 or d3 for anything, you can remove all references to them.